frontend/plan/src/reducers/schedule.js

```diff
@@ -94,15 +108,14 @@ export const schedule = (state = initialState, action) => {
                 ...state,
                 scheduleSelected: action.scheduleId,
             };
-        case ADD_SCHED_ITEM:
+        case TOGGLE_CHECK:
```

Comment:
Yeah I agree about the naming. It was initially toggle_check when the cart was handled separately in the Redux state, but now that this affects both the cart and the schedule, the name doesn't make as much sense. As far as bringing back ADD_SCHED_ITEM, I'll take a look at the code to see if making that change would increase the complexity. My reasoning for keeping this functionality inside a toggle reducer was that I wanted to avoid doing any sort of state data processing outside of reducers to the greatest extent possible.

Reply:
I think it's fine to make decisions based on state in the UI, since it makes the reducer actions that much clearer. but in the end, doesn't matter too much as long as we're consistent.